src/index.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+import './index.css';
+import App from './App';
+
+const root = ReactDOM.createRoot(document.getElementById('root'));
+root.render(
+  // <React.StrictMode>
+  <App />,
+  // {/* </React.StrictMode>, */}
```

Comment:
# Optional:
- I strongly advise you to remove commented code. I am asking this because it makes your code dirty and thus unprofessional.